css/style.css

```diff
@@ -0,0 +1,20 @@
+body, html {
+    font-size: 1.2em;
+    font-family: "Roboto";
+}
+
+h1, h2, h3, h4, h5, h6 {
+    color: #EE7552;
+}
+
+.jumbotron {
+    background-image: url("../images/party.jpg");
```

Comment:
I love this image (and favicon), it really shows the kind of party I had in mind when making this. However, the image is licensed as CC BY-SA 4.0 and I'm not 100% sure if that's compatible with this site's MIT licensing.

Also, I can't comment on binary files, but did you make the favicon yourself or did you get it somewhere? If the former, also need to check if the licenses are compatible just to cover ass.

As far as style, do you think the image could be faded a bit or maybe some kind of gradient overlay? It's a bit hard to read the header text on top of it.

Lastly, and sorry for the waterfall of comments on this one thing, do you think it's possible to reduce the size of the image? It's 3.2MB and a bit on the sluggish side to load, even running locally.

![network-load](https://cloud.githubusercontent.com/assets/1082806/20296523/3e4d5aee-aac0-11e6-8b9d-949b33ebc94d.png)